pydantic/utils.py

```diff
@@ -114,6 +135,12 @@ def almost_equal_floats(value_1: float, value_2: float, *, delta: float = 1e-8)
     return abs(value_1 - value_2) <= delta
 
 
+def copy_code(code: CodeType, **update: Any) -> CodeType:
+    if PY38:  # pragma: no cover
```

Comment:
```suggestion
    if PY38:
```
This should be tested.

Reply:
I guess I need a little help here. How do we test such things?

Reply:
Ci already runs tests for all supported versions of python which are then combined.